app/models/activity_type.rb

```diff
@@ -142,6 +142,10 @@ def count_existing_for_academic_year(school, academic_year)
     school.activities.where(activity_type: self).where(happened_on: academic_year.start_date..academic_year.end_date).count
   end
 
+  def public_type
+    'activity'
```

Comment:
maybe a symbol? (I just ended up double checking that public_type wasn't used to display text, rather than being used in an i18n key).